presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorAccessControl.java

```diff
@@ -122,6 +122,13 @@ default void checkCanShowSchemas(ConnectorSecurityContext context)
         return emptySet();
     }
 
+    /**
+     * Check if identity is allowed to execute SHOW CREATE SCHEMA.
+     *
+     * @throws io.prestosql.spi.security.AccessDeniedException if not allowed
+     */
+    void checkCanShowCreateSchema(ConnectorSecurityContext context, String schemaName);
```

Comment:
make it deny by default, see https://github.com/prestosql/presto/pull/3484

Reply:
Done.